### ISSUE TITLE
Unexpected merge of eupath_dev.owl

### ISSUE THREAD

**Issue by zhengj2007** [VEuPathDB-ontology/VEuPathDB-ontology#60]
We developed the VEuPathDB-ontology by importing external ontologies using URL and external terms in separate OWL files. The eupath_dev.owl only contains VEuPathDB ontology terms with EUPATH prefix. This approach is easy to maintain the ontology and keep consistency with any updates of external ontologies. However, current eupath_dev.owl merged all the imported external owl files. Please see: 
https://raw.githubusercontent.com/VEuPathDB-ontology/VEuPathDB-ontology/d4bd8f1730eb3f724ab7c5da69af4abf52466828/src/ontology/eupath_dev.owl
(or check out the eupath_dev.owl)

I don't know when it happened. At least in early 2019, it looks good. For example,
https://github.com/VEuPathDB-ontology/VEuPathDB-ontology/blob/fe28e62e072a6ed1db504b7ab6a0d4ea13b89c38/src/ontology/eupath_dev.owl
It still contains following owl:imports
```
        <owl:imports rdf:resource="http://purl.obolibrary.org/obo/eupath/dev/import_DOID.owl"/>
        <owl:imports rdf:resource="http://purl.obolibrary.org/obo/obi/NIAID-GSC-BRC.owl"/>
        <owl:imports rdf:resource="http://purl.obolibrary.org/obo/eupath/dev/externalByHand.owl"/>
        <owl:imports rdf:resource="http://purl.obolibrary.org/obo/eupath/dev/import_OBO.owl"/>
        <owl:imports rdf:resource="http://purl.obolibrary.org/obo/eupath/dev/import_OBO2.owl"/>
```

If the merge happened recently, we may need to revert the changes. Otherwise, we need to separate the EuPath prefix terms and imported external terms for further development.

**Comment by johnwjudkins** [comment]
I found the cause. The change happened when I merged the popbio terms into eupath_dev on March 12. Somehow these owl:imports lines did not survive the update.

**Comment by zhengj2007** [comment]
@johnwjudkins Could you please fix it? Thanks!

**Comment by johnwjudkins** [comment]
Now I see that on March 12 the owl:imports lines disappeared. But before that, import terms were asserted in both an external .owl file and the dev .owl file. So the merge of the dev file with the external .owl files happened before March 12 but after the fe28... commit from 2019 that you linked. I'm still trying to determine what happened.

**Comment by zhengj2007** [comment]
eupath_dev.owl should reference the external terms but not contains any other annotations. Otherwise, updated the import OWL files using ontoFox won't change anything. I think at least for the release you made last year when you ran the ontoFox to update the external ontology terms, the content in the eupath_dev.owl also updated. 

If too many changes made after the merging the import OWL files, revert commits may take more time than to separate the EuPath prefix terms with external terms in eupath_dev.owl. Since it is hard to explain, I can work on it using the other approach.

**Comment by zhengj2007** [comment]
Revert merged file to keep only VEuPathDB specific terms in the eupath_dev.owl. Import the external ontology terms from different OWL files:
```
<owl:imports rdf:resource="http://purl.obolibrary.org/obo/eupath/dev/externalByHand.owl"/>
<owl:imports rdf:resource="http://purl.obolibrary.org/obo/eupath/dev/import_DOID.owl"/>
<owl:imports rdf:resource="http://purl.obolibrary.org/obo/eupath/dev/import_OBO.owl"/>
<owl:imports rdf:resource="http://purl.obolibrary.org/obo/eupath/dev/import_OBO2.owl"/>
<owl:imports rdf:resource="http://purl.obolibrary.org/obo/obi/NIAID-GSC-BRC.owl"/>
```

The OBI NIAID-GSC-BRC view is used now. The view will be released with each OBI release and reflects the latest changes in the OBI. Due to updates in OBI, it may be some terms used by VEuPathDB not be included in the OBI NIAID-GSC-BRC view. 

For current updates, following classes and relations need to be imported:
Class
http://purl.obolibrary.org/obo/NCBITaxon_33277 Ancylostomatoidea

ObjectProperty
http://purl.obolibrary.org/obo/BFO_0000066 occurs in (from RO)
http://purl.obolibrary.org/obo/IAO_0000142 mentions
http://purl.obolibrary.org/obo/OBI_0000304 is_manufactured_by
http://purl.obolibrary.org/obo/RO_0002248 has active ingredient

**Comment by johnwjudkins** [comment]
I updated the import files (as we discussed) such that each ontology from which we use a large number of terms has its own file. input_OBO.txt has the rest. (Terms in the OBO2 file were moved to other files.)

Some issues remain:

- 'is about' and 'mentions' from IAO appear as both classes and object properties in import_OBO.owl.

- Some classes in import_ENVO.owl have multiple parents. (Maybe we should directly assert the parents of these classes in the input .txt file.)
